CODEOWNERS

```diff
@@ -59,7 +61,6 @@
 /connectivity/tests/health.go @cilium/sig-agent
 /connectivity/tests/host.go @cilium/sig-agent
 /connectivity/tests/ipsec_xfrm.go @cilium/ipsec
-/connectivity/tests/perfpod.go @cilium/sig-datapath
```

Comment:
This is because the file has moved to `connectivity/perf/benchmarks/netperf/`, right?

Reply:
yes, I've moved that long time ago, just didn't update CODEOWNERS at that time: https://github.com/cilium/cilium-cli/pull/2168
I'm trying to decouple perf command from connectivity test stuff, but making small steps towards that :)